backend/data_formatter.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+
+class DataFormatter:
+    def __init__(self, df_to_format: pd.DataFrame):
+        self._df = df_to_format.copy()
+
+
+    def get_formatted_df(self):
+        return self._df
+
+
+    def unbias(self):
```

Comment:
Mutated?

Reply:
I believe it doesn't since I copied if from Tara's code but in any case self._df is a copy() so its impossible to mutate anything